Code/src/main/java/nl/utwente/viskell/ghcj/GhciEvaluator.java

```
@@ -20,7 +20,7 @@ public GhciEvaluator() throws HaskellException {
 
     @Override
     protected List<String> getModules() {
-        return ImmutableList.of("Data.List", "Data.Maybe", "Data.Either", "Data.Monoid", "Control.Applicative");
+        return ImmutableList.of("Data.List", "Data.Maybe", "Data.Either", "Data.Monoid", "Control.Applicative", "Test.QuickCheck");
```

We'd probably want to warn the user, though, instead of failing silently, right?